code/modules/clothing/head/misc_special.dm

```diff
@@ -307,7 +307,7 @@
 	equip_delay_other = 140
 	var/datum/brain_trauma/mild/phobia/conspiracies/paranoia
 
-/obj/item/clothing/head/foilhat/equipped(mob/living/carbon/human/user, slot)
+/*  austation begin /obj/item/clothing/head/foilhat/equipped(mob/living/carbon/human/user, slot)
```

Comment:
put the comment above the object

Reply:
the WHOLE object?

I'm replacing the line here.
my /austation folder now has the proc.

Reply:
oh, I see